File: Migracao_JDBC/DonateBackend/src/main/java/br/fai/backend/donate/backend/main/dto/DoacaoDto.java
package br.fai.backend.donate.backend.main.dto;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@AllArgsConstructor
@NoArgsConstructor
public class DoacaoDto {
    private String data_doacao;  // ex: "2025-06-10"
    private String hora_doacao;  // ex: "14:30"
    private Integer quantidade_ml;
    private Long id_bancos_de_leite;
    private Long id_usuario;
}

File: Migracao_JDBC/donate-frontend/src/app/domain/dto/doacao.dto.ts
export interface DoacaoDto {
  id_bancos_de_leite: number;
  quantidade_ml: number;
  data_doacao: string;
  hora_doacao: string;
  id_usuario: number;  // obrigatório
  rua: string;          // novo
  numero: string;       // novo
  bairro: string;       // novo
}
